app/helpers/template_helpers.py

```diff
@@ -42,6 +42,90 @@ def get_page_header_context(language, theme):
     return context.get(theme)
 
 
+def get_footer_context(language_code, theme, base_url, contact_us_url, sign_out_url):
+    footer_urls = get_footer_urls(language_code, theme)
+
+    footer_warning = (
+        lazy_gettext(
+            f"Make sure you <a href='{sign_out_url}'>leave this page</a> or close your browser if using a shared device"
+        )
```

Comment:
This would fail to lookup the translation as the signout url would be interpolated before the translation is looked up.
```suggestion
        lazy_gettext(
            "Make sure you <a href='{sign_out_url}'>leave this page</a> or close your browser if using a shared device"
        ).format(sign_out_url=sign_out_url)
```